src/Workspaces/CSharp/Portable/Extensions/ParenthesizedExpressionSyntaxExtensions.cs

```diff
@@ -178,6 +178,12 @@ public static bool CanRemoveParentheses(this ParenthesizedExpressionSyntax node,
                 return true;
             }
 
+            // (x is Y z)   ->    x is Y z
+            if (expression.IsKind(SyntaxKind.IsPatternExpression))
+            {
+                return true;
+            }
```

Comment:
is this correct? I don't think we can always remove parens here. does this work for cases like this?
```c#
1 + (x is y z)
```
removing changes associativity here

Reply:
Indeed.  let me fixup.

Reply:
#resolved